stonesoup/feeder/filter.py

```diff
@@ -54,8 +60,18 @@ class MetadataValueFilter(MetadataReducer):
         that satisfy the operator condition (i.e. cause the
         :py:attr:`~operator` to return :code:`True`) are allowed through the
         filter.
+
+        Note
+        ====
+        * If :class:`~.GroundTruthPath` type is extended to have a metadata attribute, this class
+        will be applicable to this type.
+
     """
 
+    metadata_field = Property(
+        str,
+        doc="Field used to reduce set of detections")
+
```

Comment:
Is there a reason that `MetadataValueFilter` no longer subclasses `MetadataReducer`? Seeing as nothing else is changed, it would make sense to maintain the class hierarchy, which would also mean that `metadata_field` is inherited and doesn't mean to be redefined.

Reply:
I don't believe there is reason that I changed it. I will change it back.